xla/service/gpu/transforms/explicit_nccl_group_async_wrapper.h

```diff
@@ -0,0 +1,48 @@
+/* Copyright 2025 The OpenXLA Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef XLA_SERVICE_GPU_TRANSFORMS_NCCL_GROUP_ASYNC_WRAPPER_H_
+#define XLA_SERVICE_GPU_TRANSFORMS_NCCL_GROUP_ASYNC_WRAPPER_H_
```

Comment:
@chaserileyroberts please use `TENSORFLOW_COMPILER_XLA_SERVICE_GPU_TRANSFORMS_EXPLICIT_NCCL_GROUP_ASYNC_WRAPPER_H_` in the header guard

Reply:
Done